### ISSUE TITLE
🐛 [BUG]: Strange edge behaviour for type 'smoothstep' and 'step'

### ISSUE THREAD

**Issue by danisahni** [bcakmakoglu/vue-flow#1310]
### Is there an existing issue for this?

- [X] I have searched the existing issues and this is a new bug.

### Current Behavior

In case two nodes are aligned vertically (have equal x-coordinate) and are connected with a _smoothstep_ or _step_ edge, the edge has a strange peak:
![image](https://github.com/bcakmakoglu/vue-flow/assets/51887375/8e6376a9-93ae-4b20-b9ea-af1a46046cee)

### Expected Behavior

An edge without a peak

### Steps To Reproduce

Here a minimal example for reproduction:

```ts
<script setup lang="ts">
import { ref } from 'vue';
import { Elements, VueFlow, Position } from '@vue-flow/core';

const elements = ref<Elements>([
  {
    id: '1',
    label: 'Item 1',
    type: 'default',
    position: { x: 50, y: 50 },
    sourcePosition: Position.Bottom,
    targetPosition: Position.Top,
  },
  {
    id: '2',
    label: 'Item 2 ',
    type: 'default',
    position: { x: 300, y: 50 },
    sourcePosition: Position.Top,
    targetPosition: Position.Bottom,
  },
  {
    id: 'e1-2',
    source: '1',
    target: '2',
    type: 'smoothstep',
  },
  {
    id: 'e2-1',
    source: '2',
    target: '1',
    type: 'smoothstep',
  },
]);

</script>

<template>
  <div style="height: 500px; width: 100%">
    <VueFlow v-model="elements" :snap-to-grid="true" :snap-grid="[10, 10]">
    </VueFlow>
  </div>
</template>

```

### Anything else?

Version:
@vue-flow/core: 1.31.0

**Comment by bcakmakoglu** [comment]
Thanks for the report, I'll check it out! :)

**Comment by bcakmakoglu** [comment]
Fixed with `1.32.1`
